styles/style.css

```diff
@@ -0,0 +1,37 @@
+body{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+header{
+    text-align: center;
+    background-color: whitesmoke;
+}
+h1{
+    color:rgb(228, 82, 87)
+}
+header h2{
+    color: plum;
+}
+ul{
+    list-style-type: none;
+    
+}
+li{
+    display: inline;
+    margin: 2%;
+}
+a{
+    text-decoration: none;
+    color: rgb(66, 66, 121);
+}
+a:hover{
+    font-weight: bold;
+    color: rgb(228, 82, 87)
+}
+
+footer{
```

Comment:
Nice use of the common styles file to handle these footer properties across the pages!